tools/lint-md.js

```diff
@@ -25079,7 +25079,7 @@ camelcase.default = default_1$4;
 
 var minimist = function (args, opts) {
     if (!opts) opts = {};
-
+    
```

Comment:
Bunch of diffs that look like they just add back in whitespace. Might be worth re-trimming

Reply:
It gets auto-trimmed on landing. `rollup` leaves the space. I'm reluctant to add post-processing to the `rollup` output. I always look at the diffs for these with whitespace turned off.